src/QirRuntime/test/QIR-static/qir-test-math.qs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Testing.QIR.Math
+{
+    open Microsoft.Quantum.Intrinsic;
+
+    function TestSqrt() : Int
+    {
```

Comment:
Q# style is to have braces in the same line, as opposed to C#'s next-line style. I'd also suggest that the name "TestSqrt" sounds like a verb, whereas Q# style is for functions to be named as nouns.

Reply:
Fixed.